src/content/docs/fr/guides/routing.mdx

```diff
@@ -407,21 +407,27 @@ interface Page<T = any> {
 	total: number;
 	/** Numéro de la page actuelle, à partir de 1 */
 	currentPage: number;
-	/** Nombre d'éléments par page (par défaut : 25) */
+	/** Nombre d'éléments par page (par défaut : 10) */
 	size: number;
 	/** Nombre de la dernière page */
 	lastPage: number;
 	url: {
 		/** URL de la page actuelle */
 		current: string;
-		/** URL de la page précédente (si il y en a une) */
+		/** URL de la page précédente (s'il y en a une) */
 		prev: string | undefined;
-		/** URL de la page suivante (si il y en a une) */
+		/** URL de la page suivante (s'il y en a une) */
 		next: string | undefined;
+		/** url de la première page (si la page actuelle n'est pas la première page) */
+		first: string | undefined;
+		/** url de la page suivante (si la page actuelle n'est pas la dernière) */
+		last: string | undefined;
 	};
 }
 ```
 
+<ReadMore>En savoir plus sur [le prop `page` de pagination](/fr/reference/api-reference/#le-prop-page-de-pagination).</ReadMore>
```

Comment:
I haven't noticed it in `api-reference.mdx` but "*le prop*" sounds weird. Why not "*la prop*" instead of "*la propriété*" if we want to keep the abbreviation but this should be **la** and not **le** (in this PR and in #8922).

So, it should be:
```suggestion
<ReadMore>En savoir plus sur [la prop `page` de pagination](/fr/reference/api-reference/#la-prop-page-de-pagination).</ReadMore>
```

However, this PR might be merged before `api-reference.mdx` since this is a small one and #8922 is probably waiting for another reviewer. So I'm not sure we should use the translated anchor here. What do you think?

Reply:
> I haven't noticed it in `api-reference.mdx` but "_le prop_" sounds weird. Why not "_la prop_" instead of "_la propriété_" if we want to keep the abbreviation but this should be **la** and not **le** (in this PR and in #8922).
> 
> So, it should be:
> 
> However, this PR might be merged before `api-reference.mdx` since this is a small one and #8922 is probably waiting for another reviewer. So I'm not sure we should use the translated anchor here. What do you think?

yes, when I written this sentence, seems a little weird with `props`

Reply:
> However, this PR might be merged before api-reference.mdx since this is a small one and https://github.com/withastro/docs/pull/8922 is probably waiting for another reviewer. So I'm not sure we should use the translated anchor here. What do you think?
We can merge this one before #8922, but It won't interfere with the rest. It'll save you a second PR to change the link.